engine/src/test/scala/cromwell/CromwellTestkitSpec.scala

```diff
@@ -118,6 +119,7 @@ object CromwellTestkitSpec {
 
   val TimeoutDuration = 60 seconds
 
+  // FIXME JG: Directly called here (withTestWorkflowManagerSystem & passed to CTS) and LocalBackendSpec - it's assigning one and not using (looks like a candidate for MainSpec type behavior!)
```

Comment:
I explained last week why they'll be there. Since I'm changing nearly all meaningful tests, it's going to be impossible to stay on top of develop while I proceed.